singularity-eos/eos/eos_ideal.hpp

```diff
@@ -78,6 +78,11 @@ class IdealGas : public EosBase<IdealGas> {
       const Real rho, const Real sie, Real *lambda = nullptr) const {
     return MYMAX(0.0, _gm1 * rho * sie);
   }
+  PORTABLE_INLINE_FUNCTION Real
+  MinInternalEnergyFromDensity(const Real rho, Real *lambda = nullptr) const {
+    return 0.0;
```

Comment:
@jhp-lanl could you add the not enabled warning function here?

Reply:
This value is actually valid for the ideal gas. However, an appropriate shift was not implemented in the Shift modifier (which is the only thing that would cause this value to be non-zero). I've added that now